CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.3.0] - 2020-12-10
```

Comment:
Please update the date too, tomorrow going to merge this PR

Reply:
Hey, I'm AFK right now, could handle this in one hour

Reply:
@rddimon Changed in 238d968acabb6c5afe4a62c2430376f2ce85879e